runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/FlinkKeyUtils.java

```diff
@@ -60,4 +67,37 @@
           e);
     }
   }
+
+  /** The Coder for the Runner's encoded representation of a key. */
+  static class ByteBufferCoder extends StructuredCoder<ByteBuffer> {
```

Comment:
Not Flink specific, so maybe move to common/coders package? Perhaps such coder even already exists elsewhere?

Reply:
Don't think that it exists already because `ByteBuffer` is Flink's way of storing the encoded key. The other Runners either have built-in encoded comparison of the key, or simply do not compare encoded.